ssl/quic/quic_channel.c

```diff
@@ -1870,12 +1870,12 @@ void ossl_quic_channel_subtick(QUIC_CHANNEL *ch, QUIC_TICK_RESULT *res,
      *   - determine the time at which we should next be ticked.
      */
 
-    /* Nothing to do yet if connection has not been started. */
-    if (ch->state == QUIC_CHANNEL_STATE_IDLE)
-        return;
-
-    /* If we are in the TERMINATED state, there is nothing to do. */
-    if (ossl_quic_channel_is_terminated(ch)) {
+    /*
+     * If the connection has not yet started, or we are in the TERMINATED state,
+     * there is nothing to do.
+     */
+    if (ch->state == QUIC_CHANNEL_STATE_IDLE
```

Comment:
may be we should have a function (`ossl_quic_channel_is_idle()`) for this, just to make code consistent with style set by Hugo. This is cosmetic nit, I don't insist on.

Reply:
I don't particularly feel the need for this.